environments/staging/app-processes.yml

```diff
@@ -1,3 +1,4 @@
+formplayer_command_args: '-javaagent:/home/cchq/dd-java-agent.jar '
```

Comment:
nit: the space at the end is unintuitive. It seems like it's not actually needed either, looking at where `{{ app_proceses_config.formplayer_command_args }}` is used below